src/abstractdataframe/abstractdataframe.jl

```diff
@@ -197,18 +197,30 @@ julia> rename!(uppercase, df)
 ```
 """
 function rename!(df::AbstractDataFrame, vals::AbstractVector{Symbol};
-                 makeunique::Bool=false)
-    rename!(index(df), vals, makeunique=makeunique)
+                 makeunique::Bool=false, mergeduplicates::MergeDuplicates=nothing)
+    if !makeunique && isa(mergeduplicates, Function)
```

Comment:
the docstring seems not to have been updated.

Reply:
in particular, I am not clear what `rename!`/`rename` should do when `mergeduplicates` is passed.